os/community-platforms.md

```diff
@@ -0,0 +1,47 @@
+# Community supported platforms
+
+CoreOS Container Linux supports a growing community of open source contributors. This page lists the most fully developed integrations for Container Linux on cloud and other platforms.
+
+## Cloud providers
+
+* [AURO Cloud][auro]
+* [Brightbox Cloud][brightbox]
+* [cloud.ca][cloud-ca]
+* [Exoscale][exoscale]
+* [Ikoula][ikoula]
+* [NIFTY Cloud][nifty]
+* [Packet][packet]
+* [Rackspace Cloud][rackspace]
+* [RimuHosting LaunchtimeVPS][rimuhosting]
+* [VEXXHOST Cloud][vexxhost]
+* [Vultr VPS][vultr]
+
+## Other providers
+
+* [CloudStack][cloudstack]
+* [Eucalyptus][eucalyptus]
+* [libvirt][libvirt]
+* [QEMU][qemu]
```

Comment:
QEMU should really be considered supported.
Our docs aren't great, but it's honestly our most tested platform.

cc @crawford, is there history here?

Reply:
We should move it to a supported platform since it meets our criteria.

Reply:
moved to the Cloud Providers section of the Container Linux landing page.https://coreosdev.atlassian.net/browse/DOCS-37